### PR TITLE
BUG: add multishell and zSpace md5 test case files

### PR DESCRIPTION
**Pull request by Hui-Xie**
Add MultiShell and zSpace's test files into Slicer.kitware.com, and modify corresponding ctest files.

Passed test like below:

putamen:$ ctest
Test project /scratch/hxie1/Projects/BrainsToolsBuild/DWIConvert
      Start  1: Uncompress_DWIConvertGeSignaHdxBMatrixTest.tar.gz
 1/69 Test  #1: Uncompress_DWIConvertGeSignaHdxBMatrixTest.tar.gz ..........   Passed    0.01 sec
      Start  2: Uncompress_DWIConvertGeSignaHdxBigEndianTest.tar.gz
 2/69 Test  #2: Uncompress_DWIConvertGeSignaHdxBigEndianTest.tar.gz ........   Passed    0.01 sec
      Start  3: Uncompress_DWIConvertGeSignaHdxFSLTest.tar.gz
 3/69 Test  #3: Uncompress_DWIConvertGeSignaHdxFSLTest.tar.gz ..............   Passed    0.01 sec
      Start  4: Uncompress_DWIConvertGeSignaHdxTest.tar.gz
 4/69 Test  #4: Uncompress_DWIConvertGeSignaHdxTest.tar.gz .................   Passed    0.01 sec
      Start  5: Uncompress_DWIConvertGeSignaHdxtBMatrixTest.tar.gz
 5/69 Test  #5: Uncompress_DWIConvertGeSignaHdxtBMatrixTest.tar.gz .........   Passed    0.01 sec
      Start  6: Uncompress_DWIConvertGeSignaHdxtTest.tar.gz
 6/69 Test  #6: Uncompress_DWIConvertGeSignaHdxtTest.tar.gz ................   Passed    0.01 sec
      Start  7: Uncompress_DWIConvertPhilipsAchieva2BMatrixTest.tar.gz
 7/69 Test  #7: Uncompress_DWIConvertPhilipsAchieva2BMatrixTest.tar.gz .....   Passed    0.01 sec
      Start  8: Uncompress_DWIConvertPhilipsAchieva2Test.tar.gz
 8/69 Test  #8: Uncompress_DWIConvertPhilipsAchieva2Test.tar.gz ............   Passed    0.01 sec
      Start  9: Uncompress_DWIConvertPhilipsAchieva3Test.tar.gz
 9/69 Test  #9: Uncompress_DWIConvertPhilipsAchieva3Test.tar.gz ............   Passed    0.01 sec
      Start 10: Uncompress_DWIConvertPhilipsAchieva4Test.tar.gz
10/69 Test #10: Uncompress_DWIConvertPhilipsAchieva4Test.tar.gz ............   Passed    0.01 sec
      Start 11: Uncompress_DWIConvertPhilipsAchieva6Test.tar.gz
11/69 Test #11: Uncompress_DWIConvertPhilipsAchieva6Test.tar.gz ............   Passed    0.01 sec
      Start 12: Uncompress_DWIConvertPhilipsAchieva7Test.tar.gz
12/69 Test #12: Uncompress_DWIConvertPhilipsAchieva7Test.tar.gz ............   Passed    0.01 sec
      Start 13: Uncompress_DWIConvertPhilipsAchievaBMatrixTest.tar.gz
13/69 Test #13: Uncompress_DWIConvertPhilipsAchievaBMatrixTest.tar.gz ......   Passed    0.01 sec
      Start 14: Uncompress_DWIConvertPhilipsAchievaBigEndianTest.tar.gz
14/69 Test #14: Uncompress_DWIConvertPhilipsAchievaBigEndianTest.tar.gz ....   Passed    0.01 sec
      Start 15: Uncompress_DWIConvertPhilipsAchievaTest.tar.gz
15/69 Test #15: Uncompress_DWIConvertPhilipsAchievaTest.tar.gz .............   Passed    0.01 sec
      Start 16: Uncompress_DWIConvertSiemensTrioSyngo2004A1Test.tar.gz
16/69 Test #16: Uncompress_DWIConvertSiemensTrioSyngo2004A1Test.tar.gz .....   Passed    0.01 sec
      Start 17: Uncompress_DWIConvertSiemensTrioSyngo2004A2Test.tar.gz
17/69 Test #17: Uncompress_DWIConvertSiemensTrioSyngo2004A2Test.tar.gz .....   Passed    0.01 sec
      Start 18: Uncompress_DWIConvertSiemensTrioTim2Test.tar.gz
18/69 Test #18: Uncompress_DWIConvertSiemensTrioTim2Test.tar.gz ............   Passed    0.01 sec
      Start 19: Uncompress_DWIConvertSiemensTrioTim3Test.tar.gz
19/69 Test #19: Uncompress_DWIConvertSiemensTrioTim3Test.tar.gz ............   Passed    0.01 sec
      Start 20: Uncompress_DWIConvertSiemensTrioTimBigEndian1Test.tar.gz
20/69 Test #20: Uncompress_DWIConvertSiemensTrioTimBigEndian1Test.tar.gz ...   Passed    0.01 sec
      Start 21: Uncompress_DWIConvertSiemensTrioTimTest.tar.gz
21/69 Test #21: Uncompress_DWIConvertSiemensTrioTimTest.tar.gz .............   Passed    0.01 sec
      Start 22: Uncompress_DWIConvertSiemensVerioTest.tar.gz
22/69 Test #22: Uncompress_DWIConvertSiemensVerioTest.tar.gz ...............   Passed    0.01 sec
      Start 23: Uncompress_GeSignaHDx.tar.gz
23/69 Test #23: Uncompress_GeSignaHDx.tar.gz ...............................   Passed    1.48 sec
      Start 24: Uncompress_GeSignaHDxBigEndian.tar.gz
24/69 Test #24: Uncompress_GeSignaHDxBigEndian.tar.gz ......................   Passed    1.31 sec
      Start 25: Uncompress_GeSignaHDxt.tar.gz
25/69 Test #25: Uncompress_GeSignaHDxt.tar.gz ..............................   Passed    1.93 sec
      Start 26: Uncompress_PhilipsAchieva1.tar.gz
26/69 Test #26: Uncompress_PhilipsAchieva1.tar.gz ..........................   Passed    0.80 sec
      Start 27: Uncompress_PhilipsAchieva2.tar.gz
27/69 Test #27: Uncompress_PhilipsAchieva2.tar.gz ..........................   Passed    0.81 sec
      Start 28: Uncompress_PhilipsAchieva3.tar.gz
28/69 Test #28: Uncompress_PhilipsAchieva3.tar.gz ..........................   Passed    0.80 sec
      Start 29: Uncompress_PhilipsAchieva4.tar.gz
29/69 Test #29: Uncompress_PhilipsAchieva4.tar.gz ..........................   Passed    0.80 sec
      Start 30: Uncompress_PhilipsAchieva6.tar.gz
30/69 Test #30: Uncompress_PhilipsAchieva6.tar.gz ..........................   Passed    0.96 sec
      Start 31: Uncompress_PhilipsAchieva7.tar.gz
31/69 Test #31: Uncompress_PhilipsAchieva7.tar.gz ..........................   Passed    0.83 sec
      Start 32: Uncompress_PhilipsAchievaBigEndian1.tar.gz
32/69 Test #32: Uncompress_PhilipsAchievaBigEndian1.tar.gz .................   Passed    0.82 sec
      Start 33: Uncompress_SiemensTrio-Syngo2004A-1.tar.gz
33/69 Test #33: Uncompress_SiemensTrio-Syngo2004A-1.tar.gz .................   Passed    0.91 sec
      Start 34: Uncompress_SiemensTrio-Syngo2004A-2.tar.gz
34/69 Test #34: Uncompress_SiemensTrio-Syngo2004A-2.tar.gz .................   Passed    0.92 sec
      Start 35: Uncompress_SiemensTrioTim1.tar.gz
35/69 Test #35: Uncompress_SiemensTrioTim1.tar.gz ..........................   Passed    0.35 sec
      Start 36: Uncompress_SiemensTrioTim2.tar.gz
36/69 Test #36: Uncompress_SiemensTrioTim2.tar.gz ..........................   Passed    0.35 sec
      Start 37: Uncompress_SiemensTrioTim3.tar.gz
37/69 Test #37: Uncompress_SiemensTrioTim3.tar.gz ..........................   Passed    0.36 sec
      Start 38: Uncompress_SiemensTrioTimBigEndian1.tar.gz
38/69 Test #38: Uncompress_SiemensTrioTimBigEndian1.tar.gz .................   Passed    0.35 sec
      Start 39: Uncompress_SiemensVerio.tar.gz
39/69 Test #39: Uncompress_SiemensVerio.tar.gz .............................   Passed    0.34 sec
      Start 40: Uncompress_DWIConvertMultiShellFSITestSmallSize.tar.gz
40/69 Test #40: Uncompress_DWIConvertMultiShellFSITestSmallSize.tar.gz .....   Passed    0.02 sec
      Start 41: Uncompress_SpaceThicknessDiffDicom.tar.gz
41/69 Test #41: Uncompress_SpaceThicknessDiffDicom.tar.gz ..................   Passed    0.02 sec
      Start 42: DWIConvertDWICompareHelpTest
42/69 Test #42: DWIConvertDWICompareHelpTest ...............................   Passed    0.15 sec
      Start 43: DWIConvertGeSignaHdxTest
43/69 Test #43: DWIConvertGeSignaHdxTest ...................................   Passed   58.45 sec
      Start 44: DWIConvertGeSignaHdxFSLTest
44/69 Test #44: DWIConvertGeSignaHdxFSLTest ................................   Passed   30.23 sec
      Start 45: FSLToNrrdTest
45/69 Test #45: FSLToNrrdTest ..............................................   Passed   11.03 sec
      Start 46: NrrdToFSLTest
46/69 Test #46: NrrdToFSLTest ..............................................   Passed   39.44 sec
      Start 47: NrrdToFSLTest2
47/69 Test #47: NrrdToFSLTest2 .............................................   Passed   39.23 sec
      Start 48: DWIConvertGeSignaHdxBigEndianTest
48/69 Test #48: DWIConvertGeSignaHdxBigEndianTest ..........................   Passed   58.50 sec
      Start 49: DWIConvertGeSignaHdxBMatrixTest
49/69 Test #49: DWIConvertGeSignaHdxBMatrixTest ............................   Passed   58.57 sec
      Start 50: DWIConvertGeSignaHdxtTest
50/69 Test #50: DWIConvertGeSignaHdxtTest ..................................   Passed   91.71 sec
      Start 51: DWIConvertGeSignaHdxtBMatrixTest
51/69 Test #51: DWIConvertGeSignaHdxtBMatrixTest ...........................   Passed   92.33 sec
      Start 52: DWIConvertPhilipsAchievaTest
52/69 Test #52: DWIConvertPhilipsAchievaTest ...............................   Passed   41.50 sec
      Start 53: DWIConvertPhilipsAchievaBigEndianTest
53/69 Test #53: DWIConvertPhilipsAchievaBigEndianTest ......................   Passed   41.04 sec
      Start 54: DWIConvertPhilipsAchievaBMatrixTest
54/69 Test #54: DWIConvertPhilipsAchievaBMatrixTest ........................   Passed   41.15 sec
      Start 55: DWIConvertPhilipsAchieva2Test
55/69 Test #55: DWIConvertPhilipsAchieva2Test ..............................   Passed   40.17 sec
      Start 56: DWIConvertPhilipsAchieva2BMatrixTest
56/69 Test #56: DWIConvertPhilipsAchieva2BMatrixTest .......................   Passed   40.17 sec
      Start 57: DWIConvertPhilipsAchieva3Test
57/69 Test #57: DWIConvertPhilipsAchieva3Test ..............................   Passed   38.21 sec
      Start 58: DWIConvertPhilipsAchieva4Test
58/69 Test #58: DWIConvertPhilipsAchieva4Test ..............................   Passed   41.36 sec
      Start 59: DWIConvertPhilipsAchieva6Test
59/69 Test #59: DWIConvertPhilipsAchieva6Test ..............................   Passed   45.02 sec
      Start 60: DWIConvertPhilipsAchieva7Test
60/69 Test #60: DWIConvertPhilipsAchieva7Test ..............................   Passed   45.39 sec
      Start 61: DWIConvertSiemensTrioTimTest
61/69 Test #61: DWIConvertSiemensTrioTimTest ...............................   Passed   22.83 sec
      Start 62: DWIConvertSiemensTrioTimBigEndian1Test
62/69 Test #62: DWIConvertSiemensTrioTimBigEndian1Test .....................   Passed   22.94 sec
      Start 63: DWIConvertSiemensTrioTim2Test
63/69 Test #63: DWIConvertSiemensTrioTim2Test ..............................   Passed   23.03 sec
      Start 64: DWIConvertSiemensTrioTim3Test
64/69 Test #64: DWIConvertSiemensTrioTim3Test ..............................   Passed   22.70 sec
      Start 65: DWIConvertSiemensVerioTest
65/69 Test #65: DWIConvertSiemensVerioTest .................................   Passed   22.99 sec
      Start 66: DWIConvertSiemensTrio_Syngo2004A_1Test
66/69 Test #66: DWIConvertSiemensTrio_Syngo2004A_1Test .....................   Passed   15.52 sec
      Start 67: DWIConvertSiemensTrio_Syngo2004A_2Test
67/69 Test #67: DWIConvertSiemensTrio_Syngo2004A_2Test .....................   Passed   15.59 sec
      Start 68: SpaceThicknessDiffDicom
68/69 Test #68: SpaceThicknessDiffDicom ....................................   Passed    0.23 sec
      Start 69: DWIConvertMultiShellFSITestSmallSize
69/69 Test #69: DWIConvertMultiShellFSITestSmallSize .......................   Passed    3.70 sec

100% tests passed, 0 tests failed out of 69

Total Test time (real) = 1017.49 sec
putamen:$

### CODE REVIEW

File: DWIConvert/TestSuite/CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,11 @@
 set(DWITarDir ${TestData_DIR}/DWI_TestData_TARBALLS)
 
 #make sure: tarballs are stored in /scratch/hxie1/Projects/BrainsToolsBuild/ExternalData/TestData/DWI_TestData_TARBALLS/
+# 1 upload testcase file into BRAINSTools/Public/ in http://slicer.kitware.com;
```
Comment (hjmjohnson) [inline DWIConvert/TestSuite/CMakeLists.txt]: Good comments.

File: DWIConvert/TestSuite/CMakeLists.txt [hunk 2]
```diff
@@ -44,8 +49,9 @@ ${DWITarDir}/SiemensTrioTim2.tar.gz
 ${DWITarDir}/SiemensTrioTim3.tar.gz
 ${DWITarDir}/SiemensTrioTimBigEndian1.tar.gz
 ${DWITarDir}/SiemensVerio.tar.gz
+${DWITarDir}/DWIConvertMultiShellFSITestSmallSize.tar.gz
```
Comment (hjmjohnson) [inline DWIConvert/TestSuite/CMakeLists.txt]: Good Work!

File: DWIConvert/TestSuite/CMakeLists.txt [hunk 3]
```diff
@@ -96,7 +102,7 @@ foreach(CLP DWICompare DWISimpleCompare)
 endforeach()
 
 ## Tool for comparing if BVec and BVal files are the same.
-add_executable(BFileCompareTool BFileCompareTool.cpp)
+add_executable(BFileCompareTool BFileCompareTool.cpp chop.cpp)
```
Comment (hjmjohnson) [inline DWIConvert/TestSuite/CMakeLists.txt]: chop.cpp should not be in this line.

File: DWIConvert/TestSuite/CMakeLists.txt [hunk 4]
```diff
@@ -130,6 +136,10 @@ set(TstOutput ${CMAKE_CURRENT_BINARY_DIR})
 
 set(DWIBASELINE_DIR ${TestData_DIR}/DWI_TestData_OUTPUTS)
 
+## DUMMYPROG
```
Comment (hjmjohnson) [inline DWIConvert/TestSuite/CMakeLists.txt]: This was a one time use script.  It should not be compiled every night, or perhaps ever again.

If you wish to keep the code for historical purposes, it needs to be commented-out, or moved somewhere else.

This is probably not worth keeping around in it's current form.  Copy and paste it into one of your OneNote notebooks and keep it out of BRAINSTools

File: DWIConvert/TestSuite/chop.cpp [hunk 5]
```diff
@@ -0,0 +1,37 @@
+//
```
Comment (hjmjohnson) [inline DWIConvert/TestSuite/chop.cpp]: This file does not belong in BRAINSTools.  It was a quick one-time use script.

**Comment by Hui-Xie** [comment]
According to professor's suggestions, I deleted the one time script file. 

I did the Ctest again this morning. All 69 Ctest case passed which includes 2 new added test cases.

Changes have been submitted into this pull request.

Thanks.

**Pull request merged**